ui/platform_window/x11/x11_window_manager_ozone.cc

```diff
@@ -10,11 +10,19 @@ namespace ui {
 
 X11WindowManagerOzone::X11WindowManagerOzone() : event_grabber_(None) {}
 
-X11WindowManagerOzone::~X11WindowManagerOzone() {}
+X11WindowManagerOzone::~X11WindowManagerOzone() {
```

Comment:
The reason of having all this Add/Remove/Get is to find out which window an event originally belongs to. 
Check https://github.com/Igalia/chromium/pull/60/files/8f92a49421cdf50a883d5e8e581d7e9a4ba52726#diff-55a067ffaffdf8cea9d2308e8e680659R136

So, if one has menu opened and moves mouse over the background window, PlatformEventDispatcher checks who can dispatch event and sends that event to a foreground window (menu) with locations of the background window. Those locations must be fixed, but we have to find the exact window, where this event is originally coming from. 

As an alternative, we could keep grap as it is and have the following thing - https://cs.chromium.org/chromium/src/ui/views/widget/desktop_aura/desktop_window_tree_host_x11.cc?q=desktop_window_tree_host_x11+package:^chromium$&l=1796 and have a g_current_capture_ in platform_display_default.cc like desktop_window_tree_host_x11 has here - https://cs.chromium.org/chromium/src/ui/views/widget/desktop_aura/desktop_window_tree_host_x11.cc?l=71